entries/p/paypal.com.json

```diff
@@ -4,12 +4,13 @@
     "url": "https://www.paypal.com/",
     "tfa": [
       "sms",
-      "totp"
+      "totp",
+      "u2f",
+      "custom-software"
     ],
-    "documentation": "https://www.paypal.com/us/smarthelp/article/faq4057",
```

Comment:
This documentation page does still explain the 2FA setup process better than [faq4502](https://www.paypal.com/ee/smarthelp/article/faq4502). I think it should stay.